App/components/views/ModularView.tsx

```diff
@@ -14,7 +12,7 @@ interface Props {
 const styles = StyleSheet.create({
   container: {
     borderRadius,
-    backgroundColor: Colors.shadow,
+    backgroundColor: Colors.backgroundLight,
```

Comment:
I've been creating theme interfaces for components like this in the theme file and applying the styling from that rather than referencing the colour directly.

Reply:
Have usage sample I can take a look at?

Reply:
https://github.com/petridishdev/aries-mobile-agent-react-native/blob/86ecc889a7a01d4ea365bf2f8f5d292dd598e3c3/App/theme.ts#L66